code/utils/rf.py

```diff
@@ -0,0 +1,27 @@
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.cross_validation import KFold
+import numpy as np
+
+
+def rf_accuracy(X_train, y_train, X_test, y_test, est=1000, feat=10, depth=10):
```

Comment:
@AlonDaks it's just a default value, but I can also make the default value greater?